packages/design-system/src/styles/components/_FormLabel.scss

```diff
@@ -35,23 +35,23 @@
 }
 
 .ds-c-field__hint {
-  color: $color-muted;
+  color: $form-hint__color;
   display: block;
-  font-weight: $font-normal;
+  font-weight: $font-weight-normal;
 }
 
 .ds-c-field__hint--inverse {
-  color: $color-muted-inverse;
+  color: $form-hint__color--inverse;
 }
 
 .ds-c-field__error-message {
-  color: $color-error;
+  color: $form-field__color--error;
```

Comment:
Looking at `form-hint__color` vs `form-field__color--error`, I almost feel like the error should have the same treatment to be `form-error__color`. It's not changing the color of the form field when it's in an error state; the error message is a sub-element of the form label, just like the form hint.

Reply:
Good call! updated/tested